Minor fixes in `eo::ite` and `define` docs. (#129)
cvc5/ethos: user_manual.md

The Eunoia language contains further commands for declaring symbols that are not standard SMT-LIB version 3.0:

> - `(define <symbol> (<typed-param>*) <term> <attr>*)`, defines `<symbol>` to be a lambda term whose arguments and body are given by the command, or just an arbitrary term defined by provided the body, if the argument list is empty (i.e., it can be even a non-function term). Note that in contrast to the SMT-LIB command `define-fun`, a return type is not provided. 

Reviewer: ```suggestion
- `(define <symbol> (<typed-param>*) <term> <attr>*)`, defines `<symbol>` to be a lambda term whose arguments and body are given by the command, or just an arbitrary term defined by provided the body, if the argument list is empty (i.e., it may be a non-function term). Note that in contrast to the SMT-LIB command `define-fun`, a return type is not provided. 
```

Author: Fixed

> <!-- Are there other attributes that can be provided? -->

Reviewer: There is a possible addition `:is_eq` that I proposed in another PR.

> At present time, it is possible to provide the attribute `:type` to the definition, which instructs the checker to perform type checking on the given term (see [type checking define](#tcdefine)).

Reviewer: ```suggestion
It is possible to provide the attribute `:type` to the definition, which instructs the checker to perform type checking on the given term (see [type checking define](#tcdefine)).
```

Reviewer: "At present time" is a bit redundant.
